beacon/goclient/observability.go

```diff
@@ -0,0 +1,76 @@
+package goclient
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ssvlabs/ssv/observability"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+type beaconNodeStatus string
+
+const (
+	observabilityName      = "github.com/ssvlabs/ssv/beacon/goclient"
+	observabilityNamespace = "ssv.cl"
+
+	statusUnknown beaconNodeStatus = "unknown"
+	statusSyncing beaconNodeStatus = "syncing"
+	statusOK      beaconNodeStatus = "ok"
```

Comment:
I'd change this to `synced` since it's really the only thing we know.

Reply:
fixed